tmux-wifi.tmux

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env bash
 
 CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
-source "$CURRENT_DIR/scripts/helpers.sh"
+source "$CURRENT_DIR/scripts/source_module.sh"
```

Comment:
Sourcing `source_module.sh` here shouldn't be necessary since we're calling the files in `scripts/`. On macOS the plugin does not work for me without the line removed.

Reply:
Sure, approve the edit and we're good to go! Might I suggest removing macos from your repo name? Old repos will be redirected to the old one without issues.

Reply:
Sure thing, I approved the PR. I definitely plan on renaming the repo once everything is merged in!

Reply:
You'll still need to merge it, btw. You're the only one that can do that...